src/controller/COPFormController.ts

```diff
@@ -38,4 +46,12 @@ export class COPFormController extends BaseHttpController {
         const COPForm = await this.formService.getCOPForm(name);
         res.json(COPForm);
     }
+
+    @httpPost('/', TYPE.ProtectMiddleware, TYPE.EditMiddleware)
+    public async create(@requestBody() form: object,
```

Comment:
Just because this could be either create _or_ update, maybe this should be named similar to the service method?
```suggestion
    public async createOrUpdate(@requestBody() form: object,
```